File: solutions/2727-number-of-senior-citizens/solution.py
class Solution(object):
    def countSeniors(self, details):
        count =int(0)
        for i in range(len(details)) :
            if details[i][11]>='6' and details[i][12]>='0' :
                if details[i][11]=='6' and details[i][12]=='0' :
                    continue
                count = count+1
        return count

File: solutions/2727-number-of-senior-citizens/solution.js

var countSeniors = function(details) {
    let count=0;
   for(let i=0;i<details.length;i++){
        if(details[i][11]>=6 && details[i][12]>=0){
            if(details[i][11]==6 && details[i][12]==0){
                continue;
            }
            count++;
        }
   }
   return count;
};
